### PR TITLE
Update Settings screens

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: Student, Teacher
release note: Improved access & organization of settings

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1007393&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  </tbody>
</table>



#### Release Note: 
Improved access & organization of settings

#### Affected Apps: Student, Teacher

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.29% | 93.43% | -0.14%
Core/Core/Profile/UserProfile.swift | 0% | -- | --
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Notifications/APICommunicationChannel.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+    }
+}
+
+public enum CommunicationChannelWorkflowState: String, Codable {
+    case active, unconfirmed
+}
+
+// https://canvas.instructure.com/doc/api/communication_channels.html#method.communication_channels.index
+public struct GetCommunicationChannelsRequest: APIRequestable {
```
Comment (shichongrui) [inline Core/Core/Notifications/APICommunicationChannel.swift]: Is this the pattern we are following now? API structs and requestables in the same file?

Comment (vanwagonet) [reply Core/Core/Notifications/APICommunicationChannel.swift]: Yeah, and also putting it in the folder nearest to its use.

**Pull request merged**